documentation/src/docs/deployment/kubernetes-install.md

```diff
@@ -0,0 +1,441 @@
+      accessModes: [ "ReadWriteOnce" ]
+      storageClassName: "standard"
+      resources:
+        requests:
+          storage: 20Gi
+
+    image:
+      repository: pravega/pravega
+      tag: 0.4.0
+      pullPolicy: IfNotPresent
+
+    tier2:
+      filesystem:
+        persistentVolumeClaim:
+          claimName: pravega-tier2
+```
+
+where:
+
+- `[ZOOKEEPER_HOST]` is the host or IP address of your Zookeeper deployment.
+
+Deploy the Pravega cluster.
+
+```
+$ kubectl create -f pravega.yaml
+```
+
+Verify that the cluster instances and its components are running.
+
+```
+$ kubectl get PravegaCluster
+NAME      AGE
+pravega   27s
+```
+
+```
+$ kubectl get all -l pravega_cluster=pravega
+NAME                                DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/pravega-pravega-controller   1         1         1            1           1m
+
+NAME                                       DESIRED   CURRENT   READY     AGE
+rs/pravega-pravega-controller-7489c9776d   1         1         1         1m
+
+NAME                                DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/pravega-pravega-controller   1         1         1            1           1m
+
+NAME                                       DESIRED   CURRENT   READY     AGE
+rs/pravega-pravega-controller-7489c9776d   1         1         1         1m
+
+NAME                                DESIRED   CURRENT   AGE
+statefulsets/pravega-bookie         3         3         1m
+statefulsets/pravega-segmentstore   3         3         1m
+
+NAME                                             READY     STATUS    RESTARTS   AGE
+po/pravega-bookie-0                              1/1       Running   0          1m
+po/pravega-bookie-1                              1/1       Running   0          1m
+po/pravega-bookie-2                              1/1       Running   0          1m
+po/pravega-pravega-controller-7489c9776d-lcw9x   1/1       Running   0          1m
+po/pravega-segmentstore-0                        1/1       Running   0          1m
+po/pravega-segmentstore-1                        1/1       Running   0          1m
+po/pravega-segmentstore-2                        1/1       Running   0          1m
+
+NAME                             TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)              AGE
+svc/pravega-bookie-headless      ClusterIP   None           <none>        3181/TCP             1m
+svc/pravega-pravega-controller   ClusterIP   10.3.255.239   <none>        10080/TCP,9090/TCP   1m
+```
+
+A `PravegaCluster` instance is only accessible WITHIN the cluster (i.e. no outside access is allowed) using the following endpoint in
+the PravegaClient.
+
+```
+tcp://<cluster-name>-pravega-controller.<namespace>:9090
+```
+
+The `REST` management interface is available at:
+
+```
+http://<cluster-name>-pravega-controller.<namespace>:10080/
+```
+
+[Check this](#direct-access-to-the-cluster) to enable direct access to the cluster for development purposes.
```

Comment:
@adrianmo this reference is not mentioned "#direct-access-to-the-cluster)" i feel it has been updated as 'enable external access':
(https://github.com/pravega/pravega-operator/blob/489e407137769e7e1c63a198a3e9ffebf4adcf45/README.md#enable-external-access)
should i update or remove this....??

Reply:
@Aparnarr I updated the README on friday after mergint the upgrade feature. Please do the following:

- Remove the "Direct access to the cluster" section and links
- Add the "enable external access" section
- Add the "Upgrade a Pravega Cluster" section
- Add the "Scale a Pravega Cluster"
- Check that the rest of sections are up to date

Thanks.

Reply:
Doc up-to-date please check